### PR TITLE
Update setup docs

### PR DESCRIPTION
**Pull request by kamolhasan**

### CODE REVIEW

File: docs/setup/install/community.md [hunk 1]
```diff
@@ -178,35 +121,13 @@ $ helm repo update
...
-appscode/kubedb-catalog     {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB Catalog by AppsCode - Catalog for database versions
 
-# Step 1: Install KubeDB Community operator chart
+#  Install KubeDB Community operator chart
 $ helm template kubedb-community appscode/kubedb \
   --version {{< param "info.community" >}}       \
   --namespace kube-system                        \
-  --set-file license=/path/to/the/license.txt    \
+  --set-file global.license=/path/to/the/license.txt    \
   --set cleaner.skip=true | kubectl apply -f -
```
Comment (hossainemruz) [inline docs/setup/install/community.md:130]: Does it work? Shouldn't it be `--set kubedb-community.cleaner.skip=true` ?

File: docs/setup/install/enterprise.md [hunk 2]
```diff
@@ -230,54 +146,15 @@ $ helm repo update
...
-  --set-file license=/path/to/the/license.txt                  \
-  --set cleaner.skip=true | kubectl apply -f -
+# Install KubeDB Enterprise operator chart
+$ helm template kubedb-enterprise appscode/kubedb       \
+    --version {{< param "info.community" >}}            \
+    --namespace kube-system                             \
+    --set-file global.license=/path/to/the/license.txt  \
+    --set kubedb-enterprise.enabled=true                \
+    --set kubedb-autoscaler.enabled=true                \
+    --set cleaner.skip=true | kubectl apply -f -
```
Comment (hossainemruz) [inline docs/setup/install/enterprise.md:157]: This won't work. We have to pass `cleaner.skip` for each component chart or we have to make cleaner part a global variable.

File: docs/setup/legacy-install/community.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+```bash
+$ helm repo add appscode https://charts.appscode.com/stable/
+$ helm repo update
+$ helm search repo appscode/kubedb --version {{< param "info.community" >}}
+NAME                        CHART VERSION APP VERSION   DESCRIPTION
+appscode/kubedb             {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB by AppsCode - Production ready databases on Kubernetes
+appscode/kubedb-catalog     {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB Catalog by AppsCode - Catalog for database versions
+
+# Step 1: Install KubeDB Community operator chart
+$ helm install kubedb-community appscode/kubedb \
```
Comment (hossainemruz) [inline docs/setup/legacy-install/community.md:80]: `appscode/kubedb` > `appscode/kubedb-community`?

File: docs/setup/legacy-install/community.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+```bash
+$ helm repo add appscode https://charts.appscode.com/stable/
+$ helm repo update
+$ helm search appscode/kubedb --version {{< param "info.community" >}}
+NAME                        CHART VERSION APP VERSION   DESCRIPTION
+appscode/kubedb             {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB by AppsCode - Production ready databases on Kubernetes
+appscode/kubedb-catalog     {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB Catalog by AppsCode - Catalog for database versions
+
+# Step 1: Install KubeDB Community operator chart
+$ helm install appscode/kubedb --name kubedb-community \
```
Comment (hossainemruz) [inline docs/setup/legacy-install/community.md:132]: `appscode/kubedb` -> `appscode/kubedb-community` ?

File: docs/setup/legacy-install/community.md [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+<div class="tab-pane fade" id="script" role="tabpanel" aria-labelledby="script-tab">
+
+## Using YAML
+
+If you prefer to not use Helm, you can generate YAMLs from KubeDB chart and deploy using `kubectl`. Here we are going to show the prodecure using Helm 3.
+
+```bash
+$ helm repo add appscode https://charts.appscode.com/stable/
+$ helm repo update
+$ helm search repo appscode/kubedb --version {{< param "info.community" >}}
```
Comment (hossainemruz) [inline docs/setup/legacy-install/community.md:178]: `appscode/kubedb` > `appscode/kubedb-community`?

File: docs/setup/legacy-install/enterprise.md [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+NAME                        CHART VERSION APP VERSION   DESCRIPTION
+appscode/kubedb             {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB by AppsCode - Production ready databases on Kubernetes
+appscode/kubedb-catalog     {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB Catalog by AppsCode - Catalog for database versions
+
+$ helm search repo appscode/kubedb-enterprise --version {{< param "info.enterprise" >}}
+NAME                        CHART VERSION APP VERSION DESCRIPTION
+appscode/kubedb-enterprise  {{< param "info.enterprise" >}}  {{< param "info.enterprise" >}}  KubeDB Enterprise by AppsCode - Enterprise features for KubeDB
+
+# Step 1: Install KubeDB Community operator chart
+$ helm install kubedb-community appscode/kubedb \
```
Comment (hossainemruz) [inline docs/setup/legacy-install/enterprise.md:100]: `appscode/kubedb` -> `appscode/kubedb-community`?

File: docs/setup/legacy-install/enterprise.md [hunk 7]
```diff
@@ -0,0 +1,312 @@
...
+
+## Using Helm 2
+
+KubeDB can be installed via [Helm](https://helm.sh/) using the [chart](https://github.com/kubedb/installer/tree/{{< param "info.installer" >}}/charts/kubedb-enterprise) from [AppsCode Charts Repository](https://github.com/appscode/charts). To install the chart with the release name `kubedb`:
+
+```bash
+$ helm repo add appscode https://charts.appscode.com/stable/
+$ helm repo update
+
+$ helm search repo appscode/kubedb --version {{< param "info.community" >}}
```
Comment (hossainemruz) [inline docs/setup/legacy-install/enterprise.md:160]: `appscode/kubedb` -> `appscode/kubedb-community`?

File: docs/setup/legacy-install/enterprise.md [hunk 8]
```diff
@@ -0,0 +1,312 @@
...
+NAME                        CHART VERSION APP VERSION   DESCRIPTION
+appscode/kubedb             {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB by AppsCode - Production ready databases on Kubernetes
+appscode/kubedb-catalog     {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB Catalog by AppsCode - Catalog for database versions
+
+$ helm search repo appscode/kubedb-enterprise --version {{< param "info.enterprise" >}}
+NAME                        CHART VERSION APP VERSION DESCRIPTION
+appscode/kubedb-enterprise  {{< param "info.enterprise" >}}  {{< param "info.enterprise" >}}  KubeDB Enterprise by AppsCode - Enterprise features for KubeDB
+
+# Step 1: Install KubeDB Community operator chart
+$ helm install appscode/kubedb --name kubedb-community \
```
Comment (hossainemruz) [inline docs/setup/legacy-install/enterprise.md:170]: `appscode/kubedb` -> `appscode/kubedb-community`?

File: docs/setup/legacy-install/enterprise.md [hunk 9]
```diff
@@ -0,0 +1,312 @@
...
+
+## Using YAML
+
+If you prefer to not use Helm, you can generate YAMLs from KubeDB chart and deploy using `kubectl`. Here we are going to show the prodecure using Helm 3.
+
+```bash
+$ helm repo add appscode https://charts.appscode.com/stable/
+$ helm repo update
+
+$ helm search repo appscode/kubedb --version {{< param "info.community" >}}
```
Comment (hossainemruz) [inline docs/setup/legacy-install/enterprise.md:230]: `appscode/kubedb` -> `appscode/kubedb-community`?

File: docs/setup/legacy-install/enterprise.md [hunk 10]
```diff
@@ -0,0 +1,312 @@
...
+NAME                        CHART VERSION APP VERSION   DESCRIPTION
+appscode/kubedb             {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB by AppsCode - Production ready databases on Kubernetes
+appscode/kubedb-catalog     {{< param "info.community" >}}  {{< param "info.community" >}}  KubeDB Catalog by AppsCode - Catalog for database versions
+
+$ helm search repo appscode/kubedb-enterprise --version {{< param "info.enterprise" >}}
+NAME                        CHART VERSION APP VERSION DESCRIPTION
+appscode/kubedb-enterprise  {{< param "info.enterprise" >}}  {{< param "info.enterprise" >}}  KubeDB Enterprise by AppsCode - Enterprise features for KubeDB
+
+# Step 1: Install KubeDB Community operator chart
+$ helm template kubedb-community appscode/kubedb \
```
Comment (hossainemruz) [inline docs/setup/legacy-install/enterprise.md:240]: `appscode/kubedb` -> `appscode/kubedb-community`?

File: docs/setup/legacy-install/kubectl_plugin.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (hossainemruz) [inline docs/setup/legacy-install/kubectl_plugin.md:1]: Why this doc been moved in legacy section? Shouldn't this be in the main install section?

File: docs/setup/legacy-install/troubleshoting.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
+---
+title: Troubleshooting KubeDB Installation
```
Comment (hossainemruz) [inline docs/setup/legacy-install/troubleshoting.md:2]: Shouldn't it be in main installation section?

File: docs/setup/legacy-uninstall/community.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+</div>
+<div class="tab-pane fade" id="script" role="tabpanel" aria-labelledby="script-tab">
+
+## Using YAML (with helm 3)
+
+If you prefer to not use Helm, you can generate YAMLs from KubeDB chart and uninstall using `kubectl`.
+
+```bash
+$ helm template kubedb-community appscode/kubedb --namespace kube-system | kubectl delete -f -
```
Comment (hossainemruz) [inline docs/setup/legacy-uninstall/community.md:58]: `appscode/kubedb` -> `appscode/kubedb-community`?

File: docs/setup/legacy-uninstall/kubectl_plugin.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
+---
+title: Uninstall KubeDB kubectl Plugin
```
Comment (hossainemruz) [inline docs/setup/legacy-uninstall/kubectl_plugin.md:2]: Shouldn't be under main uninstall section?

**Review by hossainemruz — CHANGES_REQUESTED** [review]

File: docs/setup/install/community.md (hunk 1 above)
Comment (kamolhasan) [reply docs/setup/install/community.md]: fixed!

File: docs/setup/install/enterprise.md (hunk 2 above)
Comment (kamolhasan) [reply docs/setup/install/enterprise.md]: global.skipCleaner=true used!

File: docs/setup/legacy-install/community.md (hunk 4 above)
Comment (kamolhasan) [reply docs/setup/legacy-install/community.md:132]: fixed!

File: docs/setup/legacy-install/community.md (hunk 5 above)
Comment (kamolhasan) [reply docs/setup/legacy-install/community.md:178]: fixed!

File: docs/setup/legacy-install/enterprise.md (hunk 10 above)
Comment (kamolhasan) [reply docs/setup/legacy-install/enterprise.md]: fixed!

File: docs/setup/legacy-install/troubleshoting.md (hunk 12 above)
Comment (kamolhasan) [reply docs/setup/legacy-install/troubleshoting.md]: removed from here!

File: docs/setup/legacy-uninstall/community.md (hunk 13 above)
Comment (kamolhasan) [reply docs/setup/legacy-uninstall/community.md:58]: fixed!

File: docs/setup/legacy-uninstall/kubectl_plugin.md (hunk 14 above)
Comment (kamolhasan) [reply docs/setup/legacy-uninstall/kubectl_plugin.md]: removed from here!

File: docs/setup/legacy-install/kubectl_plugin.md (hunk 11 above)
Comment (kamolhasan) [reply docs/setup/legacy-install/kubectl_plugin.md]: removed from here!

File: docs/setup/legacy-install/enterprise.md (hunk 7 above)
Comment (kamolhasan) [reply docs/setup/legacy-install/enterprise.md:164]: fixed!

File: docs/setup/legacy-install/community.md (hunk 3 above)
Comment (kamolhasan) [reply docs/setup/legacy-install/community.md]: fixed!

File: docs/setup/legacy-install/community.md (hunk 4 above)
Comment (tamalsaha) [reply docs/setup/legacy-install/community.md]: Revered. I think the release name should be `kubedb`. This was users can just helm upgrade to an enterprise installation.

**Pull request merged**